specifications/config.md

```diff
@@ -0,0 +1,140 @@
+# Configuration
+
+This document specifies how Stormpath SDKs should handle configuration.
+
+## Processing
+
+The order in which the configuration should be processed.
+
+1. Load configuration according to *loading order*.
+2. Check and apply any *post-processing keys*.
+3. Validate the configuration according to the *validation rules*.
+
+## Loading order
+
+The order in which the configuration should be loaded.
+
+1. Default configuration.
+2. `apiKey.properties` file from `~/.stormpath` directory**.
+3. `stormpath.[json or yaml]` file from `~/.stormpath` directory**.
+4. `apiKey.properties` file from application directory**.
```

Comment:
Sorry, you're correct and I should have been more specific.  Yes, every process has a directory associated with where it was launched.  However, that may not mean an installation directory at all.

If I just go to `$HOME` and run a java application:

`java -jar myjar-version.jar`

The software is up and running and `System.getProperty("user.dir")` would report my $HOME directory.  That's not a likely place where Stormpath application config would be.